### PR TITLE
DR-1445: Update Skaffold template to work with modular helm charts

### PR DESCRIPTION
**Pull request by snf2ye**
Included Changes:
- Updating chart versions
- Moving to modular chart model

Test strategy: I was able to successfully deploy using `skaffold run` using the skaffold.yaml generated by this template.

### CODE REVIEW

**Review by nmalfroy — APPROVED** [review]

**Review by fboulnois — APPROVED** [review]
Are we still pinned to 1.3.1? Or can we use a newer version of Skaffold now?

**Comment by snf2ye** [comment]
> Are we still pinned to 1.3.1? Or can we use a newer version of Skaffold now?

I just tested with the latest (Skaffold v1.15.0), and it ran okay, but didn't actually point to the new container. Just to test, I reverted back to v1.3.1 and everything worked as expected.
Screen Shot 2020-10-27 at 10 08 31 AM

Also, brew no longer allows you to directly install the old version of skaffold using the raw github link, as we have listed in our "getting started" docs. I found a workaround, I'll put it in the readme for now. But, we're getting pretty far behind in skaffold versions. We should probably figure out what's causing the issue. 
image

I created a ticket to look into this further: https://broadworkbench.atlassian.net/browse/DR-1446.

**Comment by nmalfroy** [comment]
Sorry, I mean to update the workaround and I straight up forgot to :(. Did
you end up using curl to just download directly?

**Comment by nmalfroy** [comment]
Also, happy to update the readme

**Comment by snf2ye** [comment]
> Sorry, I mean to update the workaround and I straight up forgot to :(. Did you end up using curl to just download directly?

@nmalfroy I just updated it again to a curl command. Please feel free to change!

**Review by fboulnois — APPROVED** [review]
lgtm, nice workaround for the getting started doc

File: docs/jade-getting-started.md [hunk 1]
```diff
@@ -110,7 +110,8 @@ continuous development of Kubernetes resources. Newer versions are incompatible
 with our development environments, so version 1.3.1 is installed instead.
 
 ```
-brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/5db9ede616f5d681fa9873b150416d6795e0e0e9/Formula/skaffold.rb
+curl https://raw.githubusercontent.com/Homebrew/homebrew-core/5db9ede616f5d681fa9873b150416d6795e0e0e9/Formula/skaffold.rb --output skaffold.rb
```
Comment (nmalfroy) [inline docs/jade-getting-started.md:113]: Thanks!

**Review by nmalfroy — APPROVED** [review]

**Pull request merged**